src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const key of actions) {
+    for (const val in key) {
```

Comment:
```suggestion
  for (const action of actions) {
    switch(action.type) {
      case 'clear': ...
    }
```
Вот так можно использовать switch case